### DISCUSSION TITLE
Hardware flow control on Sonoff Zigbee 3.0 USB Dongle Plus

### DISCUSSION THREAD

**Discussion by PetrMa** [Koenkk/Z-Stack-firmware#465] [General]
Hi,

I would like to ask if HFC is supported on Sonoff Zigbee 3.0 USB Dongle Plus?
I changed witch position on stick to ON and changed in Z2M option to rtscts: true but after restart of Z2M I´ve got an error message:

info  16: Logging to console and directory: '/config/zigbee2mqtt/log/2023-07-16.14-36-34' filename: log.txt
info  16: Starting Zigbee2MQTT version 1.32.1 (commit #unknown)
info  16: Starting zigbee-herdsman (0.16.0)
error 16: Error while starting zigbee-herdsman
error 16: Failed to start zigbee
error 16: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
error 16: Exiting...
error 16: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:103:27)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
    at Zigbee.start (/app/lib/zigbee.ts:58:27)
    at Controller.start (/app/lib/controller.ts:101:27)
    at start (/app/index.js:107:5)

What am I doing wrong? I´am on latest firmware [CC1352P2_CC2652P_launchpad_coordinator_20230507.zip]
Thanks, regards,
Petr

**Comment by okastl** [comment]
> I would like to ask if HFC is supported on Sonoff Zigbee 3.0 USB Dongle Plus?

No, it is not.
